addons/event/models/event_event.py

```diff
@@ -106,7 +67,7 @@ def _default_description(self):
         return self.env['ir.ui.view']._render_template('event.event_default_descripton')
 
     name = fields.Char(string='Event', translate=True, required=True)
-    note = fields.Html(string='Note')
+    note = fields.Html(string='Note', store=True, compute="_compute_note", readonly=False)
```

Comment:
We could maybe have a test in synchronization test checking that note is given from type to event ? With a "false" empty type note test (something like <p><br/></p>) that should not overwrite existing content.

Reply:
Indeed, test added, you can have a look.